projects/charger_test/src/main.c

```diff
@@ -0,0 +1,123 @@
+#include <stdbool.h>
```

Comment:
I like test projects like this. naming convention for this should be something like: `test_<descriptive_test_name>`.  Over here seems like you're testing `pwm`,  `can_rx`, and `evse_connection_state`. So split it into three different projects: `test_charger_pwm`, `test_charger_mcp_rx`, and `test_charger_evse_conn`. Having these be granular will be super useful when we want to test with hardware.